src/controller/CHIPDeviceController.cpp

```diff
@@ -1001,7 +1033,7 @@ void DeviceCommissioner::OnSessionEstablishmentError(CHIP_ERROR err)
 
 void DeviceCommissioner::OnSessionEstablished()
 {
-    VerifyOrReturn(mDeviceBeingCommissioned != nullptr, OnSessionEstablishmentError(CHIP_ERROR_INVALID_DEVICE_DESCRIPTOR));
+    StopSessionIf(mDeviceBeingCommissioned == nullptr, CHIP_ERROR_INVALID_DEVICE_DESCRIPTOR);
```

Comment:
My concern here is that it's very nonobvious that `StopSessionIf` is an early return.  The old naming of VerifyOrReturn at least made that clear...

Maybe: `StopSessionAndReturnIf`?

Reply:
I have updated the code have as few changes as possible. So `StopSessionIf` is gone.